src/components/Details.svelte

```diff
@@ -74,13 +75,16 @@
             </NotEmpty>
         </div>
 
+        <MarkdownField title={$_('details.notes')} content={$details.notes}/>
```

Comment:
Who has access to set the fields passed to these instances of `<MarkdownField />` - `$details.notes`, `$details.giftcardfundraiser`, and `$details.specialoffers`? This is a potential cross-side scripting vulnerability.

Reply:
Only the people with edit access to the Google Sheet can change the fields. 
We can use `DOMPurify.sanitize` to just be safe.

Reply:
👍